openapi/components/schemas/TagUntag/TagUntagRule.yaml

```
@@ -0,0 +1,94 @@
+type: object
+description: Tag / Untag rule.
+required:
+  - name
+  - eventType
+  - addingTags
+  - removingTags
+properties:
+  id:
+    description: Unique rule identifier string.
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  name:
+    description: Rule's name.
+    type: string
+  eventType:
+    description: Rebilly event type.
+    type: string
+    enum:
+      - account-password-reset-requested
+      - account-verification-requested
+      - aml-list-possibly-matched
+      - customer-created
+      - customer-updated
+      - dispute-created
+      - dispute-modified
+      - experian-check-performed
+      - gateway-account-requested
+      - invoice-issued
+      - invoice-modified
+      - invoice-paid
+      - invoice-partially-paid
+      - invoice-partially-refunded
+      - invoice-past-due
+      - invoice-refunded
+      - invoice-revenue-recognized
+      - invoice-voided
+      - kyc-document-accepted
+      - kyc-document-rejected
+      - kyc-request-fulfilled
+      - order-completed
+      - payment-card-expired
+      - renewal-invoice-issued
+      - renewal-invoice-payment-declined
+      - subscription-activated
+      - subscription-canceled
+      - subscription-churned
+      - subscription-created
+      - subscription-ended
+      - subscription-reactivated
+      - subscription-renewed
+      - subscription-trial-ended
+      - transaction-amount-discrepancy-found
+      - transaction-declined
+      - transaction-discrepancy-found
+      - transaction-process-requested
+      - transaction-processed
+  filter:
+    description: |-
+      Filter that determines whether to send the webhook.
+      This field requires a special format.
+      Use `,` for multiple allowed values.
+      Use `;` for multiple fields.
+
+      For more information,
+      see [Using filters](https://api-reference.rebilly.com/#section/Using-filter-with-collections).
+    type: string
+    nullable: true
+  addingTags:
+    description: List of tag ID customer should be tagged with.
+    type: array
+    minItems: 0
+    maxItems: 1000
+    items:
+      type: string
+  removingTags:
+    description: List of tag ID customer should be untagged from.
```

nitpick: suggestion for readability and to remove passive 'should'
```suggestion
    description: List of tag IDs from which the customer must be untagged.
```